### ISSUE TITLE
Show building range on mouse over

### ISSUE THREAD

**Issue by lamefun** [Unvanquished/Unvanquished#637]
When you have a lot of stuff, it's hard to make anything out of all the range spheres.

**Comment by Viech** [comment]
Indeed. I have this on my list already. I want to couple it with beacons and only show range markers for blueprint buildables and those that you highlight.
